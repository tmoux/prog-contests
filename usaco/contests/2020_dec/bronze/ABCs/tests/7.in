148385247 819365820 793320372 26045448 122339799 670980573 697026021
