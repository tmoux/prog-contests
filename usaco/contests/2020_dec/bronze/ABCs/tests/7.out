26045448 122339799 670980573
